startup.py

```diff
@@ -402,8 +404,9 @@ def _default_path_software_versions(
                     # Display and version information are contained before the first ','
                     # in the output version string.
                     default_display = version_output[0:version_output.find(",")]
+
+                    # Update known oddball display values to "nicer" version numbers.
                     if "2016 Extension 2 SP1" in default_display:
```

Comment:
I am not sure we should hard code too much of this stuff here. Does it look crazy in the desktop? Any client not happy with the display can always customize it via the software entity? The less hardcoded stuff in the app, the easier it will be to maintain it :)